vpr/src/route/rr_graph_storage.h

```diff
@@ -585,7 +623,53 @@ class t_rr_graph_storage {
                             "Attempted to access RR node 'side' for non-IPIN/OPIN type '%s'",
                             rr_node_typename[node_data.type_]);
         }
-        return node_storage[id].dir_side_.side;
+        std::bitset<NUM_SIDES> side_tt = node_storage[id].dir_side_.sides;
+        for (const e_side& side : SIDES) {
+            if (side_tt[size_t(side)]) {
+                return side;
+            }
+        }
+        /* No valid sides, return an invalid value */
+        VPR_FATAL_ERROR(VPR_ERROR_ROUTE,
+                        "Invalid side for RR node '%d':\n\ttype='%s'\txlow,ylow=(%d,%d)\n\txhigh,yhigh=(%d,%d)",
```

Comment:
The error should report that the user request "node_side" on a node that has more than one side.